tpgtools/overrides/monitoring/metrics_scope.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
We'll want to delete this too, since https://github.com/GoogleCloudPlatform/magic-modules/pull/5197 will try to pull it in